helm-chart/templates/spire-agent.yaml

```diff
@@ -76,10 +79,22 @@ spec:
           while $(curl -t 'DUMMY=1' -s telnet://spire-server:8081); [ $? -ne 48 ]; do
             sleep 1
           done
+      - name: chown-volume
+        image: {{ include "ubuntu.image-server" . }}ubuntu:22.04
+        imagePullPolicy: {{ .Values.registry.imagePullPolicy }}
+        command: [ "sh", "-c", "chown -R 1000:1000 /run/spire/sockets" ]
+        volumeMounts:
+          - name: spire-agent-socket
+            mountPath: /run/spire/sockets
       containers:
       - name: spire-agent
-        image: {{ include "spire.image-server" . }}/spire-agent:1.5.4
+        image: {{ include "spire.image-server" . }}/spire-agent:1.6.1
         imagePullPolicy: {{ .Values.registry.imagePullPolicy }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          privileged: false
+          runAsUser: 1000 #spire
+          runAsNonRoot: true
```

Comment:
same questions, can this just live in the `spec` securityContext?

Reply:
I think a couple of them can be moved up, but same as above the  `allowPrivilegeEscalation:, false privileged: false` fields have to stay.